shared/src/business/entities/cases/Case.ts

```diff
@@ -1521,6 +1523,7 @@ export class Case extends JoiValidationEntity {
     this.trialLocation = undefined;
```

Comment:
I suspect `removeFromTrialWithAssociatedJudge` is redundant relative to `removeFromTrial`, unless there is a good reason for not updating the case status. This isn't relevant to this PR, but I'll investigate further